azure-pipelines.yml

```diff
@@ -22,6 +22,8 @@ trigger:
     exclude: # don't trigger if only docs and similar files changed
     - docs/*
     - '**/*.md'
+    - 'gc-azure-pipelines.yml'
```

Comment:
It would probably also be worth adding the gc-azure-pipelines.yml file to the exclude list for pr's just below here.